app/services/hyrax/versioning_service.rb

```diff
@@ -66,16 +48,8 @@ def perform_create_through_active_fedora(content, user)
         record_committer(content, user) if user
       end
 
-      def perform_create_through_valkyrie(content, user)
-        return # TODO: WINGS - Just return for now.  This method won't work until #indexing_adapter method is complete.  See issue #3800.
-        # rubocop:disable Lint/UnreachableCode
-        new_version = content.new(id: nil)
-        new_version.label = "version#{content.member_ids.length + 1}"
-        new_version = indexing_adapter.persister.save(resource: new_version)
-        content.member_ids = content.member_ids + [new_version.id]
-        content = indexing_adapter.persister.save(resource: content)
-        record_committer(content, user) if user
-        # rubocop:enable Lint/UnreachableCode
+      def perform_create_through_valkyrie(content, user) # no-op
+        raise NotImplementedError
```

Comment:
this already did nothing. better to raise an error and catch it further down than silently swallow the message.